méli-mélo/2024-04-stepsquiz/stepsquiz-limited.html

```diff
@@ -0,0 +1,100 @@
+---
+title: Steps Quiz - Content Limited Width
+dateModified: 2024-04-01
+description: "Addition of a Quiz pattern to the existing Steps Form component using the content limited width template."
+lang: en
+pageclass: cnt-wdth-lmtd
+css:
+- stepsquiz.css
+script:
+- js/stepsquiz.js
+---
+<p>Quiz pattern for the Steps Form component using the <code>panel</code> class along with the content limited width template.</p>
+
+<h2>How to use</h3>
+<ul>
+	<li>Add the CSS class name <code>cnt-wdth-lmtd</code> to the <code>&lt;body&gt;</code> of your page.</li>
+	<li>Apply <code>panel</code>, <code>panel-default</code> and <code>stepsquiz</code> CSS classes to the first <code>&lt;section&gt;</code> element of the Steps Form.</li>
+	<li>Add the <code>panel-body</code> CSS class to the <code>&lt;div&gt;</code> element containing the <code>wb-steps</code> class.</li>
+</ul>
+
+<section>
+	<h2>Working Examples</h2>
+	<ul>
+		<li><a href="index.html">Steps Quiz - Default (full width)</a></li>
+		<li><a href="#cntlmtd-1">Steps Quiz - Content Limited Width Panel</a></li>
+	</ul>
+</section>
+
+<section class="panel panel-default stepsquiz">
+	<div class="wb-frmvld provisional wb-steps quiz panel-body">
+		<h2 id="cntlmtd-1">Example - Content Limited Width</h2>
+		<form action="#" method="get" id="quiz-example">
```

Comment:
Optional:

```suggestion
		<form action="#">
```

The method `get` is implicit and I don't think that `id` is used anywhere.

Note: Probably the same fix would be required in index.html